### DISCUSSION TITLE
Idea: Make osu!mania co-op mod be able to use two different skins and  scroll speeds.

### DISCUSSION THREAD

**Discussion by Leinadix** [ppy/osu#30889] [Ideas]
Make the mania coop mode be able to use two different skins and scroll speeds, so that two different players who are using completely different skins can still play locally together.

**Comment by bdach** [comment]
Not sure if this is feasible given ideas of ditching co-op entirely are being floated around (https://github.com/ppy/osu/discussions/25936)
